_includes/chatbot-scripts.html

```diff
@@ -8,11 +8,16 @@
         rgba(0, 0, 0, 0.06) 0 1px 6px 0,
         rgba(0, 0, 0, 0.16) 0 2px 32px 0;
     border-radius: 50%;
+
+    background-image: url('/images/vica-55px-hover.svg');
```

Comment:
```suggestion
    background-image: url('/images/VICA-55px-hover.svg');
```

just in case it complains of capitalisation or whatever.